src/js/user_activity.js

```diff
@@ -0,0 +1,55 @@
+var interval = 0;
+
+function is_user_alive () {
+	if ($("body").hasClass("toc-expanded") || $("body").hasClass("sitenav-expanded")) {
+		interval = 0;
```

Comment:
Please declare interval as a var at the top of the function (var interval = 0;), javascript does weird things when changing scopes with variables not explicitly declared as such